integration-tests/cli/test/autoinstall.test.ts

```diff
@@ -49,6 +49,10 @@ test.serial(
     t.regex(stdout, /Installing packages.../);
     t.regex(stdout, /Will install @openfn\/language-common version/);
     t.regex(stdout, /Installed @openfn\/language-common@/);
+    t.regex(
+      stdout,
+      /Resolved adaptor @openfn\/language-common to version \d+(\.\d+)*/
+    );
```

Comment:
This Resolved test makes sure that linking was done successfully version is a number not string. 
This is added to an already existing test for `@latest` autoinstall